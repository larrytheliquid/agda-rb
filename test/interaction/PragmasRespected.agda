{-# OPTIONS --show-implicit #-}

module PragmasRespected where

postulate
  Foo : {A : Set₁} → Set
  Bar : Foo {A = Set}
