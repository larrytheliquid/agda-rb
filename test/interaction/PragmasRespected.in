top_command (cmd_load currentFile [])
top_command (cmd_infer_toplevel Normalised "Bar")
-- Reload to test that pragmas are respected when interface files are
-- used.
top_command (cmd_load currentFile [])
top_command (cmd_infer_toplevel Normalised "Bar")
