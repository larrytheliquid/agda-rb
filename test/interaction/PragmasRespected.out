agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-status-action "Checked,ShowImplicit")
agda2_mode_code (agda2-info-action "*All Goals*" "")
agda2_mode_code (agda2-goals-action '())
agda2_mode_code (agda2-status-action "Checked,ShowImplicit")
agda2_mode_code (agda2-info-action "*Inferred Type*" "Foo {Set}")
agda2_mode_code (agda2-goals-action '())
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-status-action "Checked,ShowImplicit")
agda2_mode_code (agda2-info-action "*All Goals*" "")
agda2_mode_code (agda2-goals-action '())
agda2_mode_code (agda2-status-action "Checked,ShowImplicit")
agda2_mode_code (agda2-info-action "*Inferred Type*" "Foo {Set}")
agda2_mode_code (agda2-goals-action '())
